### PR TITLE
Add gitter to readme

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Adds link to Gitter room dedicated to GTFS-lib discussion. Slightly awkward since this is under the TRANSIT-data-tools space but since the TRANSIT-data-tools team is the main maintainer of this repo right now, this seems ok.

### CODE REVIEW

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Is it possible to swap out the links for the `Gitter.im` ones?

**Comment by philip-cline** [comment]
The gitter ones were not working, I have moved this to matrix links which should work.

**Review by miles-grant-ibigroup — APPROVED** [review]
New link not working for me either

File: README.md [hunk 1]
```diff
@@ -14,6 +16,10 @@ The main design goals are:
 
 A gtfs-lib GTFSFeed object should faithfully represent the contents of a single GTFS feed file. At least during the initial load, no heuristics are applied to clean up the data. Basic syntax is verified, and any problems encountered are logged in detail. At this stage, fields or entites may be missing, and the data may be nonsensical. Then in an optional post-load validation phase, semantic checks are performed and problems are optionally repaired.
 
+## Getting in touch
+
+We have a Gitter [space](https://matrix.to/#/#transit-data-tools:gitter.im) for the full TRANSIT-Data-Tools project where you can post questions and comments. This includes a room dedicated to GTFS-lib discussions.
```
Comment (binh-dam-ibigroup) [inline README.md]:
```suggestion
We have a [Gitter space](https://matrix.to/#/#transit-data-tools:gitter.im) for the full TRANSIT-Data-Tools project where you can post questions and comments. This includes a room dedicated to GTFS-lib discussions.
```

**Review by binh-dam-ibigroup — APPROVED** [review]
One tiny change but otherwise good to go.

**Pull request merged**